addons/account/models/chart_template.py

```diff
@@ -314,8 +314,20 @@ def _create_bank_journals(self, company, acc_template_ref):
                 'currency_id': acc.get('currency_id', self.env['res.currency']).id,
                 'sequence': 10
             })
+
+        if company.country_id.code in self._get_countries_posting_at_bank_rec():
+            bank_journals.write({'post_at_bank_rec': True})
+
         return bank_journals
 
+    def _get_countries_posting_at_bank_rec(self):
+        """ Returns the list of the country codes of the countries for which, by default,
+        bank journals should be created so that the payments posted in them only
+        generate draft account.move objects, which get in turn posted when their
+        payment gets reconciled with a bank statement line.
+        """
+        return ['BE', 'LU']
```

Comment:
Actually I would have set the option to 'post at bank rec' inside each l10n module that needed it instead of doing it here with a list of specific country. However quentin think that it is better to either have it for everyone or not at all. So it needs to be discussed with al or avw